generators/app/tasks/appSetup/baseFilesTemplate.js

```diff
@@ -16,6 +16,7 @@ const {
   CUSTOM_BUTTON_STYLES,
   CUSTOM_TEXT_INPUT,
   CUSTOM_TEXT_INPUT_STYLES,
+  CUSTOM_TEXT_INPUT_INTERFACE,
```

Comment:
`ANIMATED_CUSTOM_TEXT_INPUT_INTERFACE`?

Reply:
No, that is the input for the normal CustomTextInput